content/organization/events/online/index.en.md

```diff
@@ -88,52 +88,52 @@ At the very bottom you should see some buttons -- click on the 'Claim Host' butt
 
 {{< figure src="enter-key.png" alt="Screenshot of the 'enter key' form in the Zoom interface" >}}
 
-### Tips for Running the Zoom meeting
+### Tips for running the Zoom meeting
 
-:eyes: After becoming Host, we suggest that you:
+The guidance in this section comes from the Baltimore, Irvine, NYC, and Santa Barbara chapter guidelines.
 
--   Enable the waiting room.
```

Comment:
We want to evaluate/confirm the order of when this needs to happen.

Reply:
Since the Zoom settings have the waiting room enabled by default, I assumed that the waiting room would be turned on automatically at the start of a meeting. However, we can discuss this at the next global team meetings.

Reply:
We don't have the waiting room enabled by default on the R-Ladies Zoom account because organisers need to enter the meeting to reclaim the host

Reply:
Hi Riva, we do have the writing room enabled by default - please see the attached screenshots, which depict the current settings in the R-Ladies Zoom account. Please note that I didn't change any of these settings - they were already like this when I looked into them.

![Screenshot 2024-09-06 124149](https://github.com/user-attachments/assets/ad1e15f3-265d-4ca3-a514-c244ba2d9dc8)
![Screenshot 2024-09-06 124128](https://github.com/user-attachments/assets/f909949f-7e71-4c5b-8e39-ef5b29245331)
![Screenshot 2024-09-06 124247](https://github.com/user-attachments/assets/88b4f886-480e-45e1-af97-e20e64518a55)
![Screenshot 2024-09-06 124234](https://github.com/user-attachments/assets/f0973738-ee92-4b53-aab0-b4070bb68e58)